Workflow/Sources/DispatchQueue+Workflow.swift

```diff
@@ -15,16 +15,7 @@
  */
 
 import Foundation
-import ReactiveSwift
 
 extension DispatchQueue {
     static let workflowExecution: DispatchQueue = .main
```

Comment:
I assume this static property is still referenced somewhere in the Workflow framework? It's marked as internal.